ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -190,6 +195,57 @@ public void SetPointerEvents(BorderedCanvas view, string pointerEventsValue)
             view.SetPointerEvents(pointerEvents);
         }
 
+        /// <summary>
+        /// Set accessibility traits for the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="accessibilityTraitsValue">Can be <see cref="JArray"/> of objects or a single object. 
+        ///     String representation of the object(s) is parsed as <see cref="AccessibilityTrait"/>.</param>
+        [ReactProp("accessibilityTraits")]
+        public void SetAccessibilityTraits(BorderedCanvas view, object accessibilityTraitsValue)
+        {
+#if WINDOWS_UWP
+            AccessibilityTrait[] result = null;
+            if (accessibilityTraitsValue != null)
+            {
+                if (accessibilityTraitsValue is JArray asJArray)
+                {
+                    var list = new List<AccessibilityTrait>();
+                    foreach (var s in asJArray.Values<string>())
+                    {
+                        if (EnumHelpers.TryParse<AccessibilityTrait>(s, out var accessibilityTrait))
```

Comment:
>TryParse [](start = 40, length = 8)

Could add a `ParseSafe` private method that returns null in for invalid values in this class and do a Linq query.
```
asJArray.Values<string>().Select(ParseSafe<AccessibilityTrait?>).OfType<AccessibilityTrait>().ToArray()
```
That's probably not any more readable/maintainable though ;-).